CoreImportExport/src/au/gov/asd/tac/constellation/importexport/delimited/translator/DatetimeAttributeTranslator.java

```diff
@@ -77,9 +77,8 @@ public PluginParameters createParameters() {
         final PluginParameter<StringParameterValue> customParam = StringParameterType.build(CUSTOM_PARAMETER_ID);
         customParam.setName("Custom Format");
         customParam.setDescription("A custom datetime format");
-        // The custom field wont enable inside the addMasterController until you
-        // click on OK and open it again. This is a work around until it can be fixed.
-        customParam.setEnabled(true);
+        // customParam should be enabled and editable if "CUSTOM" format has been specified.
+        customParam.setEnabled(formatParam.getStringValue().equals("CUSTOM"));
```

Comment:
In general it is good practice to do the equals check the other way to avoid potential null pointer errors if you have a constant so:

``"CUSTOM".equals(formatParam.getStringValue())``